lib/workers/repository/process/lookup/filter.ts

```diff
@@ -66,23 +67,43 @@ export function filterVersions(
     });
   }
 
+  const currentMajor = versioningApi.getMajor(currentVersion);
+  const currentMinor = versioningApi.getMinor(currentVersion);
+  const currentPatch = versioningApi.getPatch(currentVersion);
+
   if (allowedVersions) {
-    const isAllowedPred = getRegexPredicate(allowedVersions);
+    // allowedVersions tempalte support
+    const computedAllowedVersions = template.compile(allowedVersions, {
+      currentVersion,
+      major: currentMajor,
+      minor: currentMinor,
+      patch: currentPatch,
```

Comment:
those can be undefined or null

Reply:
I think a warning & not filtering based on `allowedVersions` will be the way to go. As handlebars will replace the `null` fields with empty string which will break the filtering and might throw errors.

What do you think, throw error or no filtering?